framework/wazuh/core/vulnerability.py

```diff
@@ -39,7 +42,7 @@ def format_fields(field_name, value):
                 except (ValueError, TypeError):
                     pass
 
-            return value
+            return value if field_name != 'external_references' else eval(value)
```

Comment:
Add an `elif` block to improve maintainability and readability. Also, try to use `json.loads` instead of `eval`. `eval` has some vulnerabilities and these references should be a valid JSON.

Reply:
Done!